quick-start.md

```diff
@@ -6,37 +6,40 @@ Here's a video that shows you all the things: https://www.youtube.com/watch?v=NX
 
 ## Prerequisites
 
```

Comment:
Let's not say "That's about it" and leave the user wondering whether they have everything covered or not. Rather be precise and they you need these things and then you are definitely good.

Reply:
Also Node was missing so i added that